http-server/server/server.go

```diff
@@ -10,18 +11,37 @@ import (
 	"time"
 )
 
-var pageAccessCount = 0
-var Param string
+// change this to use methods in the file
+type serverInfo struct {
+	pageAccessCount int
+	param           string
+}
+
+var serv = serverInfo{}
```

Comment:
`var Serv = serverInfo{}`
possibly have this public in order to use in main as `server.Serv`
then `SetParam()` can be a method as `func (si *serverInfo) SetParam(s string)`